frontend/src/style/globals.css

```diff
@@ -19,4 +19,9 @@ input[type="file"]::file-selector-button {
   border-style: solid;
   border-width: 1px;
   border-radius: 10px;
+}
+
+.help-style {
+    margin-top: 2pc;
+    padding-bottom: 10pc;
 }
```

Comment:
Add this code, will round corners for all ``<iframe>`` components so it looks a little nicer on dark mode:

```
iframe {
  border-radius: 20px;
}
```